data/coverage/2017.yml

```diff
@@ -1,6 +1,165 @@
+    title: 'Skills Matter : London Ruby User Group : This Code Sucks — A Story About
+      Nonviolent Programming'
+  choosing-ruby-for-a-new-startup:
+  - type: video
+    url: https://skillsmatter.com/skillscasts/10337-choosing-ruby-for-a-new-startup
+    title: 'Skills Matter : London Ruby User Group : Choosing Ruby for a new Startup'
+  3-ways-of-using-react-with-rails---pros-and-cons:
+  - type: video
+    url: https://skillsmatter.com/skillscasts/10338-using-react-js-with-ruby-on-rails-apps
+    title: 'Skills Matter : London Ruby User Group : 3 ways of using React with Rails
+      - pros and cons'
+may:
+  5-things-i-wish-my-grandfather-told-me-about-activerecord-and-postgres:
+  - type: video
+    url: https://skillsmatter.com/skillscasts/10184-london-ruby-may-meetup
+    title: 'Skills Matter : London Ruby User Group : 5 Things I wish my grandfather
+      told me about ActiveRecord and Postgres'
+  a-brief-(and-incomplete)-introduction-to-rust-(for-rubyists):
+  - type: video
+    url: https://skillsmatter.com/skillscasts/10207-a-brief-and-incomplete-introduction-to-rust-for-rubyists
+    title: 'Skills Matter : London Ruby User Group : A Brief (and incomplete) introduction
+      to Rust (for Rubyists)'
+  do-you-even-declarative?:
+  - type: video
+    url: https://skillsmatter.com/skillscasts/10208-do-you-even-declarative
+    title: 'Skills Matter : London Ruby User Group : Do you even declarative?'
+april:
+  mobile-journey-at-freeagent:
+  - type: video
+    url: https://skillsmatter.com/skillscasts/10028-mobile-journey-at-freeagent
+    title: 'Skills Matter : London Ruby User Group : Mobile Journey at FreeAgent'
+  practical-hypermedia-apis-in-ruby:
+  - type: video
+    url: https://skillsmatter.com/skillscasts/10029-practical-hypermedia-apis-in-ruby
+    title: 'Skills Matter : London Ruby User Group : Practical hypermedia APIs in
+      Ruby'
+  live-coding:
+  - type: video
+    url: https://skillsmatter.com/skillscasts/10051-live-coding
+    title: 'Skills Matter : London Ruby User Group : Live Coding'
+march:
+  action-cable-vs.-messagebus---live-updates-tailored-for-rails:
+  - type: video
+    url: https://skillsmatter.com/skillscasts/9862-london-ruby-march-meetup
+    title: 'Skills Matter : London Ruby User Group : Action Cable vs. MessageBus -
+      Live updates tailored for Rails '
+  the-story-of-the-not-so-dark-magic-of-application-preloading:
+  - type: video
+    url: https://skillsmatter.com/skillscasts/9917-the-story-of-the-not-so-dark-magic-of-application-preloading
+    title: 'Skills Matter : London Ruby User Group : The story of the not-so dark
+      magic of application preloading'
+  dependency-injection-with-little-boxes:
+  - type: video
+    url: https://skillsmatter.com/skillscasts/9918-dependency-injection-with-little-boxes
+    title: 'Skills Matter : London Ruby User Group : Dependency Injection with Little
+      Boxes'
+february:
+  london-ruby-lightning-talks:
```

Comment:
This one seems to be a bit of an anomaly as all of the other lightning talks are broken up into individual videos! I've added the timestamp in the titles within brackets, e.g. Base WTF (51:34), but maybe it should be formatted in a different way - let me know!

Reply:
I like the idea of the timestamp - maybe we could duplicate the video coverage for each talk and put the timestamp in the title of the coverage. e.g.:

```yaml
february:
   ch-ch-ch-ch-changes-avoiding-mutable-state-in-ruby-and-why-you-might-want-to:
   - type: video
     url: https://skillsmatter.com/skillscasts/9865-london-ruby-lightning-talks
     title: 'Skills Matter : London Ruby User Group : Ch-ch-ch-ch-Changes: Avoiding mutable state in Ruby and why you might want to (starts at 00:00)'
  a-way-to-think-about-bugs:
  - type: video
     url: https://skillsmatter.com/skillscasts/9865-london-ruby-lightning-talks
     title: 'Skills Matter : London Ruby User Group : A way to think about bugs (starts at 10:18)
  # etc...
```

This would leave the door open for people adding any other coverage (slides, etc...) for those talks (however unlikely that might be)